src/ActionModal/components/ActionModal.vue

```diff
@@ -779,7 +778,7 @@ export default {
       /* istanbul ignore next */
       update(data) {
         this.balancesLoaded = true
-        return data.balances || []
+        return JSON.parse(JSON.stringify(data.balances)) || []
```

Comment:
This works?? 👁️ I'm going to check

Reply:
Went on a hunch and worked for me. Assumed change is not triggered by the new data somehow.

Reply:
Not working for me.

Could you reproduce the bug?